requirements.txt

```diff
@@ -1 +1,2 @@
 rdflib>=6.1.1
+pandas
```

Comment:
I think you need to remove PANDAS from here too! You might have to make a requirements.optional.txt file or similar. This is because a `pip install -r requirements.txt` locally will pull PANDAS in. Sure, you could say that anyone using SPARQLWrapper from a cloned repo, rather than PIP might be ok with this, but I think the policy should be to keep required and optional dependencies in setup.py and various requirements.txt files in sync. We already have [requirements.development.txt](https://github.com/RDFLib/sparqlwrapper/blob/master/requirements.development.txt) for keeping dev requirements separate.

I think `black` should be added to requirements.development.txt?

Reply:
I forgot.

Reply:
11b674f